CONTRIBUTING.md

```diff
@@ -124,18 +125,18 @@ git push origin your-branch-name
 
 ### ✅ Good Practice
 
-- Comment on the issue to get assigned
-- Create an issue before you make a Pull Request
+*   Comment on the issue to get assigned
```

Comment:
Codacy has a fix for the issue: [Expected: 1; Actual: 3](https://app.codacy.com/gh/web3community/devprotocol.xyz/pullRequest?prid=8405536)
```suggestion
* Comment on the issue to get assigned
```